recipe/conda_build_config.yaml

```
@@ -1,4 +1,2 @@
 clang_variant:
-  - default
-  - cling_6.14.06
-  - cling_6.16.00
+  - cling_6.18.00
```

I think the default variant should be kept.
```suggestion
  - default
  - cling_6.18.00
```